### PR TITLE
Move and style Finish and Settings buttons in Javalab

### PR DESCRIPTION
**Pull request by bencodeorg**
Moves Settings and Finish buttons to the top right of Javalab, and styles them in accordance with what's been spec'd in Figma.

~~Note this is a bit of a hack in the name of getting something ready for video -- The Visualization and Editor columns are positioned absolutely (although the containing div is set up as a flex container?) I've just made the top margins of the Visualization and Editor Columns bigger to make room for a div containing the Settings and Finish buttons.~~

~~Longer term, probably worth getting the Visualization and Editor set up as flex items so that the different areas of the editor can be managed by flexbox? I also am not very experienced CSS-wise, so I may be mistaken about what to do here.~~

My most recent commit implements this using flexbox, let me know how it looks! As-is, I have the editor taking up the remaining space left to right on the screen. Looks good on a laptop, but might look kind of weird on large screens.

**Before**

image

**After**

image

## Testing story

Tested manually -- looked reasonably good at smaller viewport sizes and different zoom settings.

### CODE REVIEW

File: apps/src/javalab/JavalabSettings.jsx [hunk 1]
```diff
@@ -69,8 +69,10 @@ export class JavalabSettings extends Component {
       <div style={styles.main}>
         <JavalabButton
           icon={<FontAwesome icon="cog" />}
+          text={'Settings'}
```
Comment (maddiedierker) [inline apps/src/javalab/JavalabSettings.jsx]: nit: move this string to i18n. there's likely already a "Settings" string in common/en_us.json that you could use

Comment (molly-moen) [inline apps/src/javalab/JavalabSettings.jsx]: let's add this to i18n

File: apps/style/javalab/style.scss [hunk 2]
```diff
@@ -107,3 +107,11 @@
...
   }
 }
+
+div#visualizationResizeBar {
+  position: static;
+}
+
+#codeApp.pin_bottom {
+  top: 50px;
+}
```
Comment (molly-moen) [inline apps/style/javalab/style.scss]: nit: add blank line at end of file

**Review by molly-moen — APPROVED** [review]
Awesome, thanks for improving this!

File: apps/src/javalab/JavalabView.jsx [hunk 3]
```diff
@@ -204,11 +206,18 @@ const styles = {
   },
   buttons: {
     display: 'flex',
-    alignItems: 'center'
+    alignItems: 'center',
+    flexDirection: 'row-reverse',
```
Comment (maddiedierker) [inline apps/src/javalab/JavalabView.jsx]: why use 'row-reverse' here instead of swapping the position of the buttons in `render`?

Comment (bencodeorg) [reply apps/src/javalab/JavalabView.jsx]: Was just experimenting with flexbox today :). Switched to less confusing approach.

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**